gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/ToggleGradeItemsToolbarPanel.html

```diff
@@ -5,8 +5,16 @@
 <wicket:panel>
 
 <div id="gradeItemsToolbarItemPanel" role="menu">
-  <a href="javascript:void(0)" id="hideAllGradeItems" role="menuitem"><span wicket:message="value:label.toolbar.gradeitemhideall">Hide All</span></a>
-  <a href="javascript:void(0)" id="showAllGradeItems" role="menuitem"><span wicket:message="value:label.toolbar.gradeitemshowall">Show All</span></a>
+  <a href="javascript:void(0)" id="hideAllGradeItems" role="menuitem">
+      <span wicket:message="value:label.toolbar.gradeitemhideall">
```

Comment:
You don't need the span at all, just the <wicket:message key=.. tag.

Reply:
Remove in all the rest and good to go.

Reply:
I initially thought the span with the value attribute was used for a tooltip, which was why it was there originally. However, you are correct, this is not the case. I'll remove them accordingly.